Models/Core/Simulation.cs

```diff
@@ -107,6 +107,9 @@ public Locater Locater
             }
         }
 
+        /// <summary>Is the simulation running?</summary>
+        public bool IsRunning { get; private set; } = false;
```

Comment:
This doesn't need to be serialized, right?

```suggestion
        [JsonIgnore]
        public bool IsRunning { get; private set; } = false;
```

Reply:
It won't be serialised anyway because it has a private setter.